### ISSUE TITLE
Clan O’Conall and the Crown of the Stag

### ISSUE THREAD

**Issue by SinUsername** [skyline-emu/skyline-games-list#641]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Tested Build

1200

### Android Version

10

### SoC type

Kirin

### Device GSMArena link

https://m.gsmarena.com/huawei_p30_lite-9545.php

### Game Behaviour

It only goes all the way to the select save file menu.
![Screenshot_20220613_093345_skyline emu](https://user-images.githubusercontent.com/94080806/173378821-e1f68af9-8112-4a81-b39f-fa8d030a7de5.jpg)
![Screenshot_20220613_093358_skyline emu](https://user-images.githubusercontent.com/94080806/173378825-613cefee-4068-4936-ad2a-d1ae735e94d5.jpg)

### Is Log Level is set to 'Info'?

- [X] Yes

### Log file

[emulation.txt](https://github.com/skyline-emu/skyline-games-list/files/8891977/emulation.txt)

**Comment by Crytonics** [comment]
Incomplete log, can you retest and reupload log?

**Comment by SinUsername** [comment]
> Incomplete log, can you retest and reupload log?

No, that's what I got I checked. But I'm going to propose it again to see if it sends me complete since it has happened to me.
